spec/controllers/users_controller_spec.rb

```diff
@@ -46,6 +68,31 @@
       end
     end
 
+    context "with valid input and inviter" do
+      let(:alice) { Fabricate(:user) }
+      let(:invite_bob) { Fabricate(:invite_user, inviter: alice) }
+      let(:bob) { User.last }
```

Comment:
Oh, I see what you are doing. You are grabbing the user that is created by the `post`. This seems brittle but I guess it works.